database/scripts/lib/report_formatter.rb

```diff
@@ -9,6 +9,19 @@ module ReportFormatter
     /^nightly_|^packaging_/ => 'https://ci.ros2.org/job/',
   }
 
+  JOB_PROJECT_PATTERN = {
+    /^gz|^sdformat|^ros_gz/ => 'gazebo',
+    /^[A-Z]ci|^nightly_|^packaging_/ => 'ros',
+  }
```

Comment:
Do we still have to rely in this kind of filters? didn't we add something in the database for this?

Reply:
IIRC we have something similar in some database columns. And that might require a lot of joins with statuses. I'll address that with https://github.com/osrf/buildfarm-tools/issues/68